### ISSUE TITLE
HTC VIVE projection distance

### ISSUE THREAD

**Issue by JustNobody2017** [ohgodhowdidthis/trance#11]
Hi,
I expirience the strong problem, of the spirals not lining up. Left and right Image won't focus on one point on the screen. The eye distance adjustment, that is new in 0.3.3. won't do anything. Please help!

**Comment by ohgodhowdidthis** [comment]
Even if you set eye distance to zero the spirals don't line up? (That should show the exact same image on both eyes.) I'll look into it if that's the case.

**Comment by JustNobody2017** [comment]
Yes, even if i set the eye distance to zero. I tried a negative value just for the lulz, But it didn't work. Then I tried zero distance in the slider above and everything except the spiral disapeared.
